files/hola_player/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "npm",
+  "name": "@hola.org%2fhola_player",
```

Comment:
` npm info @hola.org%2fhola_player` gives me an error. Can you double check this